rollup.config.js

```diff
@@ -4,6 +4,8 @@ import copy from 'rollup-plugin-copy';
 import license from 'rollup-plugin-license';
 import scss from 'rollup-plugin-scss';
 import visualizer from 'rollup-plugin-visualizer';
+import postcss from 'rollup-plugin-postcss';
+import copyAssets from 'postcss-copy-assets';
```

Comment:
We end up using [`postcss-url`](https://github.com/webex/widgets/blob/master/rollup.config.js#L6) because `postcss-copy-assets` is not maintained